Modules/Core/Transform/include/itkComposeScaleSkewVersor3DTransform.h

```diff
@@ -0,0 +1,196 @@
+  using JacobianPositionType = typename Superclass::JacobianPositionType;
+  using InverseJacobianPositionType = typename Superclass::InverseJacobianPositionType;
+  using ScalarType = typename Superclass::ScalarType;
+  using InputPointType = typename Superclass::InputPointType;
+  using OutputPointType = typename Superclass::OutputPointType;
+  using InputVectorType = typename Superclass::InputVectorType;
+  using OutputVectorType = typename Superclass::OutputVectorType;
+  using InputVnlVectorType = typename Superclass::InputVnlVectorType;
+  using OutputVnlVectorType = typename Superclass::OutputVnlVectorType;
+  using InputCovariantVectorType = typename Superclass::InputCovariantVectorType;
+  using OutputCovariantVectorType = typename Superclass::OutputCovariantVectorType;
+  using MatrixType = typename Superclass::MatrixType;
+  using InverseMatrixType = typename Superclass::InverseMatrixType;
+  using CenterType = typename Superclass::CenterType;
+  using OffsetType = typename Superclass::OffsetType;
+  using TranslationType = typename Superclass::TranslationType;
+
+  using VersorType = typename Superclass::VersorType;
+  using AxisType = typename Superclass::AxisType;
+  using AngleType = typename Superclass::AngleType;
+
+  /** Scale & Skew Vector Type. */
+  using ScaleVectorType = Vector<TParametersValueType, 3>;
+  using SkewVectorType = Vector<TParametersValueType, 3>;
+
+  using ScaleVectorValueType = typename ScaleVectorType::ValueType;
+  using SkewVectorValueType = typename SkewVectorType::ValueType;
+  using TranslationValueType = typename TranslationType::ValueType;
+
+  using AxisValueType = typename Superclass::AxisValueType;
+  using ParametersValueType = typename Superclass::ParametersValueType;
+
+  /** Directly set the matrix of the transform.
+   *
+   * Orthogonality testing is bypassed in this case.
+   *
+   * \sa MatrixOffsetTransformBase::SetMatrix() */
+  void
+  SetMatrix(const MatrixType & matrix) override;
+  void
+  SetMatrix(const MatrixType & matrix, const TParametersValueType tolerance) override;
+
+  /** Set the transformation from a container of parameters
+   * This is typically used by optimizers.
+   * There are 12 parameters:
+   *   0-2   versor (right part)
+   *   3-5   translation
+   *   6-8   Scale
+   *   9-11  Skew
+   **  */
+  void
+  SetParameters(const ParametersType & parameters) override;
+
+  const ParametersType &
+  GetParameters() const override;
+
+  void
+  SetScale(const ScaleVectorType & scale);
+
+  itkGetConstReferenceMacro(Scale, ScaleVectorType);
+
+  void
+  SetSkew(const SkewVectorType & skew);
+
+  itkGetConstReferenceMacro(Skew, SkewVectorType);
+
+  void
+  SetIdentity() override;
+
+  /** This method computes the Jacobian matrix of the transformation.
+   * given point or vector, returning the transformed point or
+   * vector. The rank of the Jacobian will also indicate if the
+   * transform is invertible at this point. */
```

Comment:
Change the comment to indicate that this is not implemented and an exception will be thrown.

Reply:
Done.